src/app/containers/Navigation/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from 'react';
+import Navigation, {
+  NavigationUl,
+  NavigationLi,
+} from '@bbc/psammead-navigation';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+const NavigationContainer = () => {
+  const { script, translations, navigation } = useContext(ServiceContext);
+  const { currentPage, skipLinkText } = translations;
+
+  if (!navigation || !navigation.length) {
```

Comment:
Is this checking if `navigation` has items in it? Is the intention not to render when we have no items in `navigation`?